### PR TITLE
Removes some unused rock sprites

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Removes some unused rock sprites, which also were fully duplicate with one another, having no name

## Why It's Good For The Game
Unused, they're not up to standards and the duplicate icon names can confuse some tools we have like the dmi deconflicter

## Images of changes
Deleted the file containing this
image

## Testing
Compiled

## Changelog
NPFC

### CODE REVIEW

File: code/game/objects/structures/flora.dm [hunk 1]
```diff
@@ -319,37 +319,6 @@
...
-//a rock is flora according to where the icon file is
-//and now these defines
-/obj/structure/flora/rock
-	name = "rock"
-	desc = "A rock."
-	icon_state = "rock1"
-	icon = 'icons/obj/flora/rocks.dmi'
-	resistance_flags = FIRE_PROOF
-	anchored = TRUE
-
```
Comment (Sirryan2002) [inline code/game/objects/structures/flora.dm:331]: it would appear that these are still used

Comment (Henri215) [reply code/game/objects/structures/flora.dm:331]: Looks like it, yeah

**Comment by dj-34** [comment]
but... there's rock and rockpile object

**Comment by DGamerL** [comment]
> but... there's rock and rockpile object

My IDE didn't find those because a setting was on that I didn't notice, it also compiled with no errors so I just didn't notice it

**Comment by DGamerL** [comment]
Just gonna make a new PR for this

**Pull request closed without merging**